src/data/cassandra/Schema.hpp

```diff
@@ -484,7 +484,32 @@ class Schema {
                 qualifiedTableName(settingsProvider_.get(), "objects")
             ));
         }();
-
+        PreparedStatement selectLastTwoObjects = [this]() {
+            return handle_.get().prepare(fmt::format(
+                R"(
+                SELECT object, sequence 
+                  FROM {}               
+                 WHERE key = ?
+                   AND sequence <= ?
+              ORDER BY sequence DESC 
+                 LIMIT 2
+                )",
+                qualifiedTableName(settingsProvider_.get(), "objects")
+            ));
+        }();
+        PreparedStatement selectLastTwoObjects = [this]() {
+            return handle_.get().prepare(fmt::format(
+                R"(
+                SELECT object, sequence 
+                  FROM {}               
+                 WHERE key = ?
+                   AND sequence <= ?
+              ORDER BY sequence DESC 
+                 LIMIT 2
+                )",
+                qualifiedTableName(settingsProvider_.get(), "objects")
+            ));
+        }();
```

Comment:
why are there two of the same query?

Reply:
Hmm not sure why it's duplicated, just remove the additional one :(